packages/gatsby/cache-dir/static-entry.js

```diff
@@ -287,7 +287,13 @@ export default async function staticPage({
             onAllReady() {
               pipe(writableStream)
             },
-            onError() {},
+            onError(error) {
+              console.error(
+                `The page "${pagePath}" threw an error while rendering:\n`,
+                error
+              )
+              process.exit()
```

Comment:
I think we should `throw` instead of exiting process - this will most often be executed in child process - randomly killing child process make it difficult for parent process to know what's going on.

Throwing will bubble up error to parrent process and let main process display error and shutdown entire build gracefully

Reply:
We also rely on throwing for reach@17, so that should make it consistent